inc/TrkInfo.hh

```diff
@@ -7,21 +7,50 @@
 #ifndef TrkInfo_HH
 #define TrkInfo_HH
 #include "Offline/DataProducts/inc/GenVector.hh"
+#include "Offline/KinKalGeom/inc/SurfaceId.hh"
 #include "Offline/MCDataProducts/inc/MCRelationship.hh"
 #include "Rtypes.h"
 namespace mu2e
 {
-  // information about the track fit at a particular place.
+  // generic information about the track fit at a particular place/time
   struct TrkFitInfo {
     XYZVectorF mom;
     XYZVectorF pos;
-    float momerr = -1000;
-    float d0 = 0;
+    double time;
+    float momerr = -1000;  // projected error on the scalar momentum
+    bool inbounds = false;
+    bool gap = false;
+    bool early = false;
+    bool late = false;
+    int sid = SurfaceIdEnum::unknown;
+    int sindex = 0;
```

Comment:
Could you add comments to these variables? I think I understand inbounds, early, and late but I'm not sure what gap is